addOns/spiderAjax/src/main/resources/org/zaproxy/zap/extension/spiderAjax/resources/Messages.properties

```diff
@@ -1,8 +1,35 @@
 # This file defines the default (English) variants of all of the internationalised messages of the plugin
 
-spiderajax.api.action.scan = Runs the spider against the given URL and/or context, optionally, spidering everything in scope. The parameter 'contextName' can be used to constrain the scan to a Context, the option 'in scope' is ignored if a context was also specified. The parameter 'subtreeOnly' allows to restrict the spider under a site's subtree (using the specified 'url').
-spiderajax.api.action.scanAsUser = Runs the spider from the perspective of a User, obtained using the given context name and user name. The parameter 'url' allows to specify the starting point for the spider, otherwise it's used an existing URL from the context (if any). The parameter 'subtreeOnly' allows to restrict the spider under a site's subtree (using the specified 'url').
-
+# Api Views
+spiderajax.api.view.fullResults = Shows the full spidered content detected by the AjaxSpider.
```

Comment:
- For consistency we should use `spider` or `AJAX Spider` instead of `AjaxSpider`. The last part seems redundant though, the API already implies that the results are from the spider?
 - It might worth being more explicit and say what the actual content is (e.g. URLs in/out of scope and errors).
 - In other API endpoints it's mainly used `Gets`, there's also `Returns` and at least one that uses `Shows`... we should pick one and normalise.